pages/home-page.tsx

```diff
@@ -1,9 +1,42 @@
 import { attributes, html } from "../_content/home-page.md";
 import { Page } from "../components/layout/page";
 import { PageContent } from "../components/layout/page-content";
+import config from "../lib/config";
+import { bannerBlurData, imgixLoader } from "../lib/imgix-loader";
+
+import Image from "next/image";
+import Link from "next/link";
+import { NextSeo } from "next-seo";
+
+const meta = {
+  title: `${config.site_name} | ${config.site_tagline}`,
+};
 
 const HomePage = () => (
   <Page>
+    <NextSeo title={meta.title} titleTemplate="%s" />
+    <header>
+      <div className="max-w-xl mx-auto">
+        <h1 className="p-0">
+          <Link href="/">
+            <a>
+              <Image
+                alt="Warga Bantu Warga"
+                blurDataURL={bannerBlurData}
+                height={287}
+                layout="responsive"
+                loader={imgixLoader}
+                placeholder="blur"
+                priority={true}
+                quality={70}
+                src="hero_banner.png"
+                width={640}
+              />
+            </a>
+          </Link>
+        </h1>
+      </div>
+    </header>
     <PageContent>
       <article className="prose prose-indigo">
         <h1>{attributes.title}</h1>
```

Comment:
```suggestion
```

We could also remove this now that the header is copied over here.

Reply:
Yes, I was working on it. Updated now.